source/connect/connection-targets.txt

```diff
@@ -0,0 +1,167 @@
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, settings, client, load balancing, srv, dns
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+ 
+In this guide, you can learn how to use a connection string and ``MongoClient`` object
+to connect to different types of MongoDB deployments.
+
+Atlas
+-----
+
+To connect to a MongoDB deployment on Atlas, include the following elements
+in your connection string:
+
+- URL of your Atlas cluster
+- MongoDB username
+- MongoDB password
+
+Then, pass your connection string to the ``MongoClient`` constructor.
+
+.. tip::
+
+   Follow the :atlas:`Atlas driver connection guide </driver-connection?tck=docs_driver_python>`
+   to retrieve your connection string.
+
+When you connect to Atlas, we recommend using the {+stable-api+} client option to avoid
+breaking changes when Atlas upgrades to a new version of {+mdb-server+}.
+To learn more about the {+stable-api+} feature, see the :ref:`<csharp-stable-api>` guide.
+
+The following code shows how to use {+driver-short+} to connect to an Atlas cluster. The
+code also uses the ``server_api`` option to specify a {+stable-api+} version.
+
+.. literalinclude:: /includes/fundamentals/code-examples/connection/AtlasConnection.cs
+   :language: csharp
+   :start-after: // start atlas connection
+   :end-before: // end atlas connection
+
+Local Deployments
+-----------------
+
+To connect to a local MongoDB deployment, use ``localhost`` as the hostname. By
+default, the ``mongod`` process runs on port 27017, though you can customize this for
+your deployment.
+
+The following code shows how to use {+driver-short+} to connect to a local MongoDB
+deployment:
+
+.. literalinclude:: /includes/fundamentals/code-examples/connection/LocalConnection.cs
+   :language: csharp
+   :start-after: // start local connection
+   :end-before: // end local connection
+
+Replica Sets
+------------
+
+To connect to a replica set, specify the hostnames (or IP addresses) and 
+port numbers of the replica set members in your connection string.
+
+The following code shows how to use {+driver-short+} to connect to a replica set
+that contains three hosts:
+
+.. literalinclude:: /includes/fundamentals/code-examples/connection/ReplicaSetConnection.cs
+   :language: csharp
+   :start-after: // start-replica-set-connection-list
+   :end-before: // end-replica-set-connection-list
+
+If you aren't able to provide a full list of hosts in the replica set, you can 
+specify one or more of the hosts in the replica set and instruct {+driver-short+} to 
+perform automatic discovery to find the others. To instruct the driver to perform
+automatic discovery, perform one of the following actions:
+
+- Specify the name of the replica set as the value of the ``replicaSet`` parameter.
+- Specify ``false`` as the value of the ``directConnection`` parameter.
```

Comment:
I would say:
Specify ``false`` or omit the ``directConnection`` parameter.

Because false is the default value for directConnection